README.md

```diff
@@ -176,3 +176,14 @@ defmodule ExampleLoginHandler do
   end
 end
 ```
+
+## Projects using ExIrc (in the wild!)
+
+Below is a list of projects that we know of (if we've missed anything,
+send a PR!) that use ExIrc in the wild.
+
+1. [Kuma][kuma] by @ryanwinchester
+- [Offension][offension] by @shymega
```

Comment:
You've got mixed list types here, can you make them all unordered? You can also add [hedwig_irc](https://github.com/jeffweiss/hedwig_irc) to the list :)